README.md

```diff
@@ -512,6 +518,27 @@ The `React\Promise\PromisorInterface` provides a common interface for objects
 that provide a promise. `React\Promise\Deferred` implements it, but since it
 is part of the public API anyone can implement it.
 
+### AsyncInterop\Promise compatibility
+
+The [`PromiseInterface`](#promiseinterface) extends the
+[`AsyncInterop\Promise`](https://github.com/async-interop/promise) interface and
+thereby all promise implementations from React/Promise implement its `when()`
+method.
+
+The `AsyncInterop\Promise` interface is intended for interoperability with other
+libraries and their combinator and conversion functions.
+
+Note, that although the interface is named *Promise*, it has nothing to do with
+promises commonly known from the
+[CommonJS Promises/A](http://wiki.commonjs.org/wiki/Promises/A),
+[Promises/A+](https://promisesaplus.com) or
+[ECMAScript 2015 (ES6)](http://www.ecma-international.org/ecma-262/6.0/#sec-promise-constructor)
+specifications.
+
+If you're using React/Promise as your primary value placeholder implementation, 
+it is recommended to **not** use the `when()` method directly but the methods
+provided by the [`PromiseInterface`](#promiseinterface).
```

Comment:
Tbh, the two examples are not *exactly* the same as we don't have something like a [try() (or attempt())](http://bluebirdjs.com/docs/api/promise.try.html) function (yet).

```php
React\Promise\attempt('operation')
    ->then('transform')
    ->then('print_r')
    ->otherwise('handleError') // catch
    ->always('cleanup') // finally
    ->done() // bubble out fatal exceptions thrown from handleError or cleanup
;
```